carbon-design-system/carbon
# fix(a11y): fixing Selectable Tile HCM accessibility

Closes #6895

Fixing Selectable Tile focus state and icons indicating a selection in Hight Contrast Mode

#### Changelog

**Changed**

Selectable Tile component

#### Testing / Reviewing

On a Windows 10 machine, go to `settings --> ease of access --> high contrast` and flick the switch. Compare the storybook in Edge (Should be in High Contrast) to Chrome (Should look normal). 
For **Firefox**, you'll need to open it up and type `about:config` into the address bar. Then, search for `layout.css.prefers-contrast` and set it to `true`.

- Visit [Tile Multiselect](http://192.168.1.158:9000/?path=/story/tile--multi-select) component
- Verify that the icon shows when selecting a tile
- Verify the focus state works on the tile
